### ISSUE TITLE
Misleading Heading

### ISSUE THREAD

**Issue by Air-Git** [MicrosoftDocs/OfficeDocs-SkypeForBusiness#4478]
The first sub-heading says: "to federate with Office 365". The details enable federation with ANY third party, and not specifically with "Office 365". Furthermore, they enable users of Skype for Business on premises to connect remotely into internal users.
If an organisation does not allow federation with third parties, what specific federation needs to be enabled for the Office 365 tenant?

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 1d7fa7af-e864-84fe-68d4-b9f5e2ed0114
* Version Independent ID: 3bbc6d87-c3dd-4208-cec2-f574a997d329
* Content: [Configure Skype for Business hybrid - Skype for Business Hybrid](https://docs.microsoft.com/en-us/skypeforbusiness/hybrid/configure-federation-with-skype-for-business-online)
* Content Source: [Skype/SfbHybrid/hybrid/configure-federation-with-skype-for-business-online.md](https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/blob/live/Skype/SfbHybrid/hybrid/configure-federation-with-skype-for-business-online.md)
* Product: **skype-for-business-itpro**
* GitHub Login: @CarolynRowe
* Microsoft Alias: **crowe**

**Comment by scanum** [comment]
@Air-Git  Thank you for submitting feedback and contributing to the docs. We are currently investigating this.

**Comment by VLG17** [comment]
@Air-Git thank you for providing this feedback. I created a PR for it. Related to your last question, i added a note to [this article](https://docs.microsoft.com/en-us/skypeforbusiness/manage/federation-and-external-access/federation-support/configuring-federation-support) which described what's needed to federate with an Office 365 tenant. Hope this is enough.

I also changed the values of the parameters to Boolean values, because they were incorrect.
